pkg/systemd/services/timer-dialog.jsx

```diff
@@ -253,15 +265,46 @@ const CreateTimerDialogBody = ({ owner }) => {
                                         isOpen={isSpecificTimeOpen} setIsOpen={setSpecificTimeOpen}
                                         menuAppendTo={() => document.body} time={specificTime} is24Hour onChange={setSpecificTime} />
                         </FormGroup>}
+                        {repeat == "custom" &&
+                        <FormGroup
+                            label={_("Custom")}
+                            validated={submitted && validateCustom ? "error" : "default"}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17592-20220809-115752-7fb44acf-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)